TwitchLib.Client/Manager/JoinedChannelManager.cs

```diff
@@ -21,9 +21,8 @@ public void AddJoinedChannel(JoinedChannel joinedChannel)
 
         public JoinedChannel GetJoinedChannel(string channel)
         {
-            var success = _joinedChannels.TryGetValue(channel, out var joinedChannel);
-
-            return success ? joinedChannel : null;
```

Comment:
Not technically part of this PR, but I'm betting the default value of `JoinedChannel` is `null`. Thus there's also no reason for a ternary here. Returning `joinedChannel` will return `null` if `success` is `false`.

Reply:
I agree  I just added to discord

```
_joinedChannels.TryGetValue(channel, out var joinedChannel);
return joinedChannel;
```